### DISCUSSION TITLE
Is there a way to programatically create an admin user?

### DISCUSSION THREAD

**Discussion by nikolaydyankov** [pocketbase/pocketbase#2507] [Q&A] [answered]
My idea is to automate the deployment as much as possible. Is there an API like `pb.settings.update()` that allows you to create an admin user?

**Comment by nikolaydyankov** [comment, marked as answer]
Nevermind, I'm blind: https://pocketbase.io/docs/api-admins/

**Reply by ganigeorgiev** [reply]
Yes, the web APIs are one way to do it.

The upcoming v0.16.0 release will come also with `admin` console command:

```sh
./pocketbase admin create test@example.com 1234567890
```

Alternatively, you can also create the admin inside a migration (Go or JS). For JS migration example, you can check https://github.com/pocketbase/pocketbase/discussions/1354.
